Formula/ice.rb

```diff
@@ -25,12 +25,14 @@ class Ice < Formula
 
   option "with-additional-compilers", "Build additional Slice compilers (slice2py, slice2js, slice2rb)"
   option "with-java", "Build Ice for Java and the IceGrid GUI app"
+  option "without-python", "Build without Ice for Python"
   option "without-xcode-sdk", "Build without the Xcode SDK for iOS development (includes static libs)"
 
   depends_on "mcpp"
   depends_on "lmdb"
   depends_on :java => ["1.8+", :optional]
   depends_on :macos => :mavericks
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
```

Comment:
I think this can just be
```
  depends_on :python if MacOS.version <= :snow_leopard
```
because it's not actually adding any dependencies.

Reply:
Fixed.